### ISSUE TITLE
Add option to visit site through ipfs link

### ISSUE THREAD

**Issue by retotrinkler** [melonproject/decentralized-portal#34]
To reduce trust user needs to have in us

**Comment by jennazenk** [comment]
yep totally agree, though of displaying the IPFS hash somewhere so that users can save it if they want and access through ipfs gateway!

**Comment by jennazenk** [comment]
and specify in blog/twitter etc that portal can be access w IPFS hash

**Comment by retotrinkler** [comment]
also something like:

Hosted with ❤️ on IPFS

would be nice - maybe bottom left/right of page

**Comment by jennazenk** [comment]
<3 it ! Will do

**Comment by jennazenk** [comment]
@ferranbt
